packages/react-vapor/src/components/form/Form.tsx

```diff
@@ -15,7 +15,7 @@ export const Form: React.FunctionComponent<IFormProps> = ({children, className,
         submitEvent.preventDefault();
     };
     return (
-        <form onSubmit={onSubmit}>
+        <form onSubmit={onSubmit} style={{display: 'contents'}}>

```

Comment:
Not sure I am willing to remove all forms from the accessibility tree 😬 

> Current implementations in most browsers will remove from the accessibility tree any element with a display value of contents (but descendants will remain). This will cause the element itself to no longer be announced by screen reading technology.

https://developer.mozilla.org/en-US/docs/Web/CSS/display#display_contents

Reply:
OK for used CSS class `full-content`